sections/04_study.md

```diff
@@ -105,7 +105,81 @@ learning can address is poised to improve.
 
 ### Splicing
 
-*A separate section from general gene expression section above.*
+Pre-mRNA transcripts can be spliced into different isoforms by retaining or
+skipping subsets of exons, or including parts of introns. This alternative
+splicing provides cells with enormous spatiotemporal flexibility to generate
+multiple distinct proteins from a single gene. Splicing is catalyzed by small
+nuclear RNAs (snRNAs) and spliceosomal proteins, which detect sequence motifs
+such as splice sites and exon sequence enhancers and silencers (ESE and ESS).
+Various RNA-binding proteins and noncoding RNAs can bias these reactions by
+altering binding affinities, blocking splice sites, or sequestering splicing
+factors. This remarkable complexity lends itself to defects that underlie many
+diseases [@tag:Scotti2016_missplicing]. For instance, in Becker muscular
+dystrophy, a point mutation in dystrophin creates an ESS that induces skipping
+of exon 31. A recent study found that quantitative trait loci (QTLs) that affect
+splicing in lymphoblastoid cell lines are enriched within risk loci for
+schizophrenia, multiple sclerosis and other immune diseases, implicating
+mis-splicing as a much more widespread feature of human pathologies than
+previously thought [@tag:Li2016_variation].
+
+Sequencing studies routinely return thousands of unannotated variants. Which
+cause functional changes in splicing, and if so, how? Prediction of a “splicing
+code” has been a holy grail over the past decade. Initial machine learning
+approaches used a naive Bayes model and a 2-layer Bayesian neural network with
+thousands of hand-derived sequence-based features to predict the probability of
+exon skipping [@tag:Barash2010_splicing_code @tag:Xiong2011_bayesian]. With the
+advent of deep learning, more complex models were built that provided
+significantly better performance [@tag:Xiong2015_splicing_code
+@tag:Jha2017_integrative_models]. Importantly, these new approaches can take in
+not only genomic features, but also tissue identity and CLIP-seq measurements of
+interactions between splicing factors and RNA, which all improve predictive
+accuracy.
+
+The massive improvement seen with deep learning seems to stem from hidden layers
+being able to create new higher-order “features”, whereas earlier approaches
+often assumed independence of features and were unable to generalize.
+Higher-order understanding is especially important in splicing, which depends
+not only on the primary sequence, but also local RNA structure, tissue identity,
+splicing factor binding, and other currently unknown factors — all of which
+interact in complex, incompletely characterized ways. With new tools to
+interpret these meta-features, a major focus of current deep learning research,
+we will soon have the ability to extract a more nuanced biological understanding
+of splicing — perhaps by building different deep neural networks for different
```

Comment:
I'm curious about the idea of building different networks for different conditions.  Is there something condition-specific that could not be learned from other types of input data?  Does the cell line embedding idea used in #258 have any relevance here?

I'm not disagreeing with the sentiment, but I found it thought-provoking and was hoping you could elaborate.